regulations/static/regulations/js/source/views/comment-view.js

```diff
@@ -93,7 +94,15 @@ var CommentView = Backbone.View.extend({
         };
       }).get()
     };
-    window.localStorage.setItem(this.key, JSON.stringify(payload));
+
+    if (payload.comment) {
+      window.localStorage.setItem(this.key, JSON.stringify(payload));
+    }
+    else {
+      // don't keep comment in localstorage if saved comment is empty
+      window.localStorage.removeItem(this.key);
```

Comment:
Should this just be `this.clear()`? I don't know if the events there are important.